cartridge-cli.lua

```diff
@@ -608,14 +608,20 @@ local function which(binary)
 end
 
 local function call(command, ...)
-    local res, err = io.popen(string.format(command, ...))
+    local cmd = string.format(command, ...)
+    local res, err = io.popen(string.format('(%s) && echo OK', cmd))
```

Comment:
For what?

Reply:
OK, I got you. Can you fetch the command result as checking the `res` variable?

Reply:
No.